content/docs/user-guide/troubleshooting.md

```diff
@@ -53,3 +53,17 @@ Unable to detect supported link types, as the
 [cache directory](/doc/command-reference/config#cache) doesn't exist. It is
 usually created automatically by DVC commands that need it, but you can create
 it manually (e.g. `mkdir .dvc/cache`) to enable this check.
+
+## Checkout failed with `store: false` outputs {#checkout-no-store}
+
+DVC is unable to checkout any output that is not present in the
+<abbr>cache</abbr>.This can usually be solved by fetching from
+[import storage](/doc/command-reference/remote). However, If `store: false`
+is used in a `.dvc` file, `dvc pull` and `dvc fetch` won't attempt to retrieve the
+data from a DVC remote.
+
+To solve this, you must provide the file locally.
```

Comment:
Is it possible to suggest more elaborate possible solution(s) that will cover most of the cases? Not sure what the options are but off the top of my head: push the data from the original repo; remove store: false and try fetch/pull again; re-import; what else? If you can just write them here I'll copy edit it as needed. Thanks!